packages/space-opera/src/components/materials_panel/materials_panel.ts

```diff
@@ -27,40 +27,39 @@ import '@polymer/paper-item';
 import '@polymer/paper-slider';
 import '@material/mwc-icon-button';
 
-import {ALPHA_BLEND_MODES, checkFinite, DEFAULT_EMISSIVE_FACTOR} from '@google/model-viewer-editing-adapter/lib/main.js'
-import {createSafeObjectUrlFromUnsafe, SafeObjectUrl} from '@google/model-viewer-editing-adapter/lib/util/create_object_url.js'
 import {RGB, RGBA} from '@google/model-viewer/lib/model-viewer';
-import {customElement, html, internalProperty, property, query} from 'lit-element';
+import {customElement, html, internalProperty, query} from 'lit-element';
 import * as color from 'ts-closure-library/lib/color/color';  // from //third_party/javascript/closure/color
 
+import {TextureInfo} from '../../../../model-viewer/lib/features/scene-graph/texture-info.js';
```

Comment:
@timmmeh Probably not worth looking at the diff since I pretty much rewrote this file. Still, it'd be great if you can look over this whole file a bit, since this is where my entire usage of your API lives.